MonoGame.Framework.Content.Pipeline/Graphics/MeshHelper.cs

```diff
@@ -95,9 +95,9 @@ public static void CalculateNormals(GeometryContent geom, bool overwriteExisting
                     // by Shuangshuang Jin, Robert R. Lewis, David West.
                     //
 
-                    normals[positionIndices[ia]] += faceNormal;
```

Comment:
@grasshopperhamburger  what makes you say that `positionIndices` is the parent positions?

If that is correct then certainly this code was wrong and this is the right fix.